triggers/tests/BUILD.bazel

```diff
@@ -128,7 +128,8 @@ EOF
                     "RunnerSpec",
                     "UnfoldStateSpec",
                     "ConfigSpec",
-                ] + (["DevOnly"] if lf_version else [])
+                    "InterfaceSpec",
+                ]
```

Comment:
As triggers currently have no dev only code, we need to ensure our interfaces run as part of the normal build